### ISSUE TITLE
default settings

### ISSUE THREAD

**Issue by Hackatoan** [meeb/tubesync#521]
would be cool if there was an aera somewhere to set default settings for sources. 

example, you want to download music from a few artist

instead of changing settings each time you would only need to select mp3, metadata, etc only once

**Comment by meeb** [comment]
Hi. Is this a duplicate of #28 ?

**Comment by Hackatoan** [comment]
yeah i think it is.
